configuration/pih/htmlforms/section-maternal-danger-signs.xml

```diff
@@ -78,6 +78,7 @@
 
     <section id="danger-signs" sectionTag="section" headerStyle="title" headerCode="pihcore.mch.maternalDangerSigns.title">
 
+        <excludeIf velocityTest="$encounter.encounterType.uuid == '0CF4717A-479F-4349-AE6F-8602E2AA41D3'">
```

Comment:
Nitpick, but we should intending all tags "nested" under this (and in other places as well)

Reply:
Thanks @mogoodrich , Would you please provide an example of nesting the tags?

Reply:
@jmbabazi sorry, I should have been clearer, I just meant that <fieldset> should be indented, ie:

`<excludeIf... 
       <fieldset...
`